### PR TITLE
Added unit test to exercise bulk import range sanity check code #1618

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

**Comment by keith-turner** [comment]
@cradal  would you be interested in looking over this?  Its a follow up to #1637

**Comment by cradal** [comment]
@keith-turner Sure.

File: core/src/test/java/org/apache/accumulo/core/clientImpl/bulk/BulkImportTest.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import java.util.TreeMap;
+
+import org.apache.accumulo.core.clientImpl.bulk.Bulk.FileInfo;
+import org.apache.accumulo.core.clientImpl.bulk.Bulk.Files;
+import org.apache.accumulo.core.data.TableId;
+import org.apache.accumulo.core.dataImpl.KeyExtent;
+import org.apache.hadoop.io.Text;
+import org.junit.Assert;
+import org.junit.Test;
+
```
Comment (cradal) [inline core/src/test/java/org/apache/accumulo/core/clientImpl/bulk/BulkImportTest.java:33]: @keith-turner Try adding this line. Looks like the checkstyle is execting Assert methods to be static imports.

import static org.junit.Assert.assertEquals;

**Comment by cradal** [comment]
@keith-turner The test runs fine. Looks good.

File: core/src/test/java/org/apache/accumulo/core/clientImpl/bulk/BulkImportTest.java (hunk 1 above)
Comment (keith-turner) [reply core/src/test/java/org/apache/accumulo/core/clientImpl/bulk/BulkImportTest.java:33]: I updated to static imports in bc6fdc1

**Pull request merged**